src/Appwrite/Utopia/Response/Model/Specification.php

```diff
@@ -0,0 +1,64 @@
+<?php
+
+namespace Appwrite\Utopia\Response\Model;
+
+use Appwrite\Utopia\Response;
+use Appwrite\Utopia\Response\Model;
+
+class Specification extends Model
+{
+    public function __construct()
+    {
+
+        $this->addRule('memory', [
+            'type' => self::TYPE_INTEGER,
+            'description' => 'Memory size in MB.',
+            'default' => 0,
+            'example' => 512
+        ]);
+        $this->addRule('cpus', [
+            'type' => self::TYPE_INTEGER,
+            'description' => 'Number of CPUs.',
+            'default' => 0,
+            'example' => 1
+        ]);
+        $this->addRule('enabled', [
+            'type' => self::TYPE_BOOLEAN,
+            'description' => 'Is size enabled.',
+            'default' => false,
+            'example' => true
+        ]);
+        $this->addRule('slug', [
+            'type' => self::TYPE_STRING,
+            'description' => 'Size slug.',
+            'default' => '',
+            'example' => 's-1vcpu-512mb'
+        ]);
+        $this->addRule('plan', [
+            'type' => self::TYPE_STRING,
+            'description' => 'Size plan.',
+            'default' => '',
+            'example' => 'starter'
+        ]);
```

Comment:
I dont think this belong to CE version

Reply:
How do we separate this correctly on the cloud and generate a correct SDK?